packages/asset-service/src/service/AssetService.ts

```diff
@@ -64,4 +64,13 @@ export class AssetService {
       ? this.flatAssetData.filter((asset) => asset.network == network)
       : this.flatAssetData
   }
+
+  async description(name: string, tokenId?: string): Promise<string> {
+    const isToken = !!tokenId
+    const contractUrl = isToken ? `contract/${tokenId?.toLowerCase()}` : ''
+    const { data } = await axios.get(
+      `https://api.coingecko.com/api/v3/coins/${name.toLowerCase()}/${contractUrl}`
+    )
+    return data.description.en
```

Comment:
This needs to be wrapped in a try/catch and return something like null if there's no description.

change the return type to `Promise<string | null>`, add a `try/catch` with `return null`.

You also aren't verifying that `data` actually contains a `description` object. It's a third party, we should never assume what data is coming back.